### PR TITLE
Order lookup

### PR DESCRIPTION
**Pull request by ATGE**

### CODE REVIEW

**Comment by allmightyspiff** [comment]
## TESTING
- ✔️ `slcli --format=table order lookup 66033262 --details`
- ✔️ `slcli --format=table order lookup 66033262`

File: SoftLayer/CLI/order/lookup.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    ordered_by = "IBM"
+    user = order.get('userRecord', None)
+    if user:
+        ordered_by = "{} ({})".format(user.get('displayName'), utils.lookup(user, 'userStatus', 'name'))
+    table.add_row(['Ordered By', ordered_by])
+
+    table.add_row(['Create Date', utils.clean_time(order.get('createDate'), date_format, date_format)])
+    table.add_row(['Modify Date', utils.clean_time(order.get('modifyDate'), date_format, date_format)])
+    table.add_row(['Order Approval Date', utils.clean_time(order.get('orderApprovalDate'), date_format, date_format)])
+    table.add_row(['status', order.get('status')])
```
Comment (allmightyspiff) [inline SoftLayer/CLI/order/lookup.py:53]: Capitalize `Status`

File: SoftLayer/CLI/order/lookup.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    table.add_row(['Order Total Amount', "{price:.2f}".format(price=float(order.get('orderTotalAmount', '0')))])
+    table.add_row(['Invoice Total Amount', "{price:.2f}".
+                  format(price=float(order.get('initialInvoice', {}).get('invoiceTotalAmount', '0')))])
+
+    items = order.get('items', [])
```
Comment (allmightyspiff) [inline SoftLayer/CLI/order/lookup.py:58]: Capitalize `Items`

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, thanks.

Also, can you update the docblock for `slcli account orders` to mention this new command.

From :
```
"""Order list account."""
```

To:
"""Lists recent orders. Use `slcli order lookup <ID>` to find more details about a specific order"""

or something like that.

File: SoftLayer/CLI/order/lookup.py (hunk 1 above)
Comment (ATGE) [reply SoftLayer/CLI/order/lookup.py]: Done

File: SoftLayer/CLI/order/lookup.py (hunk 2 above)
Comment (ATGE) [reply SoftLayer/CLI/order/lookup.py:58]: Done

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**